config/config.inc.php

```diff
@@ -221,7 +221,12 @@
 if (isset($cookie->id_lang) && $cookie->id_lang) {
     $language = new Language($cookie->id_lang);
 }
-if (!isset($language) || !Validate::isLoadedObject($language) || !$language->isAssociatedToShop()) {
+
+$use_default_language = !isset($language) ||                    // if language not defined
+    !Validate::isLoadedObject($language) ||                     // or if language object is not valide
+    ( !$language->isAssociatedToShop() && !isset($employee) );  // or if ( language is not associated to shop [FO] AND employee not defined [BO] )
+
```

Comment:
I suggest to use variables name instead comments, something like : 

```php
$isNotValidLanguage = !isset($language) || !Validate::isLoadedObject($language);
$isAssociatedToShop = !$language->isAssociatedToShop();
$isEmployeeUndefined = !Validate::isLoadedObjet($employee);

$use_default_language = $isNotValidLanguage || ($isAssociatedToShop && $isEmployeeUndefined);

```

Reply:
Looks good for me ^^
Let me change this :)